packages/pigeon/example/app/windows/runner/messages.g.cpp

```diff
@@ -271,17 +271,31 @@ void MessageFlutterApi::FlutterMethod(
   EncodableValue encoded_api_arguments = EncodableValue(EncodableList{
       a_string_arg ? EncodableValue(*a_string_arg) : EncodableValue(),
   });
-  channel->Send(
-      encoded_api_arguments,
-      [on_success = std::move(on_success), on_error = std::move(on_error)](
-          const uint8_t* reply, size_t reply_size) {
-        std::unique_ptr<EncodableValue> response =
-            GetCodec().DecodeMessage(reply, reply_size);
-        const auto& encodable_return_value = *response;
+  channel->Send(encoded_api_arguments, [on_success = std::move(on_success),
+                                        on_error = std::move(on_error)](
+                                           const uint8_t* reply,
+                                           size_t reply_size) {
+    std::unique_ptr<EncodableValue> response =
```

Comment:
I think we could do the same (extract the entire implementation of the lambda to a static helper method) if we make it templated in the return value type and then add template specializations for special cases like enums. But we can leave that to a follow-up.